Cargo.toml

```diff
@@ -602,6 +601,7 @@ prometheus-closure-metric = { path = "crates/prometheus-closure-metric" }
 shared-crypto = { path = "crates/shared-crypto" }
 simulacrum = { path = "crates/simulacrum" }
 pera = { path = "crates/pera" }
+dwallet-mpc = { path = "crates/dwallet-mpc-centralized-party" }
```

Comment:
???

Reply:
This dependency was not configured correctly as a member of this project, this line fixes this.